papers/MeerTRAP/incoherent.py

```diff
@@ -0,0 +1,267 @@
+""" 
+This script creates a redshifty comparison figure of MeerTRAP,
+ASKAP/CRACO (estimates), DSA, and CHIME
+
+
+"""
+import os
+
+from astropy.cosmology import Planck18
+from zdm import cosmology as cos
+from zdm import misc_functions
+from zdm import parameters
+from zdm import survey
+from zdm import pcosmic
+from zdm import iteration as it
+from zdm import loading
+from zdm import io
+from zdm import optical as opt
+
+import numpy as np
+from zdm import survey
+from matplotlib import pyplot as plt
+from pkg_resources import resource_filename
+
+import matplotlib
+import cmasher as cmr
+from astropy import units
+
+
+defaultsize=12
+ds=4
+font = {
+        # 'family' : 'Helvetica',
+        'weight' : 'normal',
+        'size'   : defaultsize}
+matplotlib.rc('font', **font)
+
+
+def main():
+    
+    # in case you wish to switch to another output directory
+    
+    opdir='Incoherent/'
+    
+    # approximate best-fit values from recent analysis
+    # best-fit from Jordan et al
+    if True:
```

Comment:
this should propagate to the inner code base somehow.
  
Maybe, the call `state = parameters.State()`
  should now accept an optional reference, e.g. Hoffman+2025

Reply:
@profxj  Yes. This is something that I think Jordan and I should implement: whenever we publish something, we record both best-fit state parameters as an optional argument, and record a large sample of walkers to be randomly sampled from.